sw/airborne/modules/computer_vision/cv.c

```diff
@@ -191,12 +191,33 @@ void cv_run_device(struct video_config_t *device, struct image_t *img)
     }
 
     if (listener->async != NULL) {
+
+#ifdef BOARD_PARROT_MINIDRONE
+//The camera of a Parrot Minidrone can only be set (AFAIK) to output YUYV not the image routine used default UYVY
```

Comment:
it would be better to have flag setting the output format (so not related to minidrones) and force it to the correct value in the board header if relevant